packages/core/src/RenderingEngine/VolumeViewport.ts

```diff
@@ -383,52 +379,31 @@ class VolumeViewport extends Viewport implements IVolumeViewport {
   };
 
   /**
-   * Sets the slab thickness to all the volume actors in the viewport.
+   * It sets the slabThickness of the actors of the viewport. If actorUIDs are
+   * provided, it will only set the slabThickness of the actors with the given
+   * uid, but if actorUIDs are not provided, it will set the slabThickness of
+   * all the actors in the viewport.
    *
-   * @param slabThickness - The slab thickness to set.
+   * @param slabThickness - number -The slab thickness to set.
```

Comment:
We don't need `number` here right, A) the docs should use the types defined in the funciton definition when parsing typescript, B) its not formatted in jsdoc style anyway.